apps/web/src/pages/TalentNominations/NominationGroup/components/NominatorList.tsx

```diff
@@ -0,0 +1,45 @@
+import { useIntl } from "react-intl";
+
+import { NominationGroupSidebarFragment as NominationGroupSidebarFragmentType } from "@gc-digital-talent/graphql";
+import { insertBetween } from "@gc-digital-talent/helpers";
+
+import { getFullNameLabel } from "~/utils/nameUtils";
+
+interface NominatorListProps {
+  talentNominations: NominationGroupSidebarFragmentType["nominations"];
+}
+
+const NominatorList = ({ talentNominations }: NominatorListProps) => {
+  const intl = useIntl();
+
+  const nominationsSortedByNominator = (talentNominations ?? []).sort(
+    (a, b) => {
+      return (
+        (a.nominator?.lastName ?? "").localeCompare(
+          b.nominator?.lastName ?? "",
+        ) ||
+        (a.nominator?.firstName ?? "").localeCompare(
+          b.nominator?.firstName ?? "",
+        )
+      );
+    },
+  );
+  const nominatorsList = nominationsSortedByNominator.map((nomination) => (
+    <span key={nomination.id}>
+      {getFullNameLabel(
+        nomination.nominator?.firstName,
+        nomination.nominator?.lastName,
+        intl,
+      )}
+    </span>
+  ));
+  const nominatorListCommaSeparated = insertBetween(
+    // eslint-disable-next-line formatjs/no-literal-string-in-jsx
+    <span>, </span>,
+    nominatorsList,
+  );
+
+  return <>{nominatorListCommaSeparated}</>;
+};
```

Comment:
Sorry, I just happened to notice this has a non-unique key error while checking the colour contrast :sweat_smile:  I tihnk it is the `<span>` in the `insertBetween` I think the following the is esame but without the error :thinking: 

```tsx
return talentNominations.sor(...).flatMap((...) => getFullNameLabel(...)).join(", ");
```

Reply:
This might be a little tricky 🤔 
It was intentionally set-up that way as dialogs are getting plugged into that component. The spans around the name to be substituted with dialog triggers later. So the comma in spans acts as separators between dialog triggers 

![image](https://github.com/user-attachments/assets/7e4c5e8a-fcdb-45f2-9d1b-3249c2d9387e)

Reply:
Wait, this isn't a table though. It's this bit

![image](https://github.com/user-attachments/assets/42ee7caa-8a50-4bd3-98ab-89d9e3f86c9c)

![image](https://github.com/user-attachments/assets/5a72fbaf-5192-40fc-8d49-4457174fe5d3)

Reply:
ha! Right! I thought i was reviewing the nominatioons table again. My brain is mush from all these high priority tickets coming in :melting_face: 

Anyway, we do need a unique key somehow :thinking: I have no idea how to solve this with the comma separated list though :cry:

Reply:
Not the best, but gets the job done for now 🤷 https://github.com/GCTC-NTGC/gc-digital-talent/pull/13100/commits/8c5557c523b5124e742bb758b884607276ff8f46
If you log the values then example keys

![image](https://github.com/user-attachments/assets/c0866ece-1957-4caf-8bfa-0fe295db89ca)